Makefile

```diff
@@ -46,6 +46,7 @@ help:
 
 .PHONY: test
 test:
+	@echo uname: '$(shell uname)'
```

Comment:
Why did you feel this would beneficial?

Reply:
Hi and thanks for the question. Namely, while i was wrestling with github pipelines for Linux and Macos, i found this additional pipeline output very useful, especially in PR #639. So i thought i should leave it in this PR as well. There is no other value to it otherwise.